runhouse/servers/cluster_servlet.py

```diff
@@ -330,6 +336,37 @@ async def acheck_cluster_logs(self, interval_size: int):
 
         return logs_den_resp, new_start_log_line, new_end_log_line
 
+    async def aperiodic_autostop_check(self):
+        """Periodically check the autostop of the cluster"""
+        while True:
+            should_update_autostop: bool = self.autostop_helper is not None
+            cluster_config = await self.aget_cluster_config()
+
+            # Note: for now using the same interval for both autostop and status checks
+            interval_size = cluster_config.get(
```

Comment:
it is possible interval size in the config is -1 right? and thoughts on just hardcoding this to 60 (s)?

Reply:
yea good point, i think for now easiest to just hard code it until we have a good reason to make it configurable